app/client/components/cancel/CancellationSwitchConfirmed.tsx

```diff
@@ -48,6 +44,265 @@ const CancellationSwitchConfirmed = () => {
+							2022.
+						</span>
+					</li>
+					<li
+						css={css`
+							${textSans.medium()};
+							margin-bottom: ${space[4]}px;
+							line-height: 20px;
+						`}
+					>
+						<span
+							css={css`
+								margin-left: ${space[1]}px;
+							`}
+						>
+							You can manage your subscription online from your{' '}
+							<Link to="/">
+								<span
+									css={css`
+										color: ${brand[500]};
+										text-decoration: underline;
+									`}
+								>
+									Account overview.
+								</span>
+							</Link>
+						</span>
+					</li>
+				</ul>
+			</div>
+
+			<div
+				css={css`
+					border: 1px solid ${neutral[86]};
+					margin: ${space[6]}px 0;
+
+					${minWidth.tablet} {
+						margin: ${space[6]}px 0 ${space[9]}px;
+					}
+				`}
+			>
+				<div
+					css={css`
+						display: flex;
+						justify-content: space-between;
+						align-items: start;
+						background-color: ${brand[400]};
+
+						${minWidth.mobileLandscape} {
+							align-items: center;
+						}
+					`}
+				>
+					<h2
+						css={css`
+							${headline.xsmall({ fontWeight: 'bold' })};
+							margin: 0;
+							padding: ${space[3]}px;
+							color: ${brandAlt[400]};
+							${maxWidth.mobileLandscape} {
+								padding: ${space[3]}px;
+							}
+							${minWidth.tablet} {
+								font-size: 20px;
+								padding: ${space[3]}px ${space[5]}px;
+							}
+						`}
+					>
+						Download the Live app now!
+					</h2>
+				</div>
+
+				<div
+					css={css`
+						background-color: ${neutral[97]};
+						padding: ${space[6]}px ${space[5]}px;
+					`}
+				>
+					<div
+						css={css`
+							display: flex;
+							margin-bottom: ${space[4]}px;
+						`}
+					>
+						<a
+							target="_blank"
+							rel="noreferrer"
+							href={
+								'https://apps.apple.com/us/app/the-guardian/id409128287'
+							}
+							aria-label="Click to download the Guardian Daily app on the Apple App Store"
+						>
+							{/* had to add max-width in the cssOverrides as well as the sizes attribute as the attribute only doesn't work in IE11 */}
+							<GridImage
+								cssOverrides={css`
+									max-width: 119px;
+									margin-right: ${space[4]}px;
+								`}
+								imgType="png"
+								altText="googlePlay"
+								sizes="119px"
+								gridId="appleAppStore"
+								srcSizes={[140, 500]}
+							/>
+						</a>
+						<a
+							target="_blank"
+							href="https://play.google.com/store/apps/details?id=com.guardian"
+							rel="noreferrer"
+							aria-label="Click to download the Guardian Live app on Google Play"
+						>
+							<GridImage
+								cssOverrides={css`
+									max-width: 119px;
+								`}
+								imgType="png"
+								altText="googlePlay"
+								sizes="119px"
+								gridId="googlePlay"
+								srcSizes={[140, 500]}
+							/>
+						</a>
+					</div>
+					<p
+						css={css`
+							${textSans.medium()};
+							${minWidth.tablet} {
+								max-width: 70%;
+							}
```

Comment:
Not a big deal, but you could use `ch` units here and get rid of the media query completely. Something like: `max-width: 60ch`.

https://ishadeed.com/article/min-max-css/#max-width-and-the-ch-unit